### ISSUE TITLE
CC0003 is shown even for catches which end with throw

### ISSUE THREAD

**Issue by Duetting** [code-cracker/code-cracker#626]
Imho CC0003 shouldn't be shown for catch blocks which end with throw:
```csharp
                var disposable = new DisposableObject();
                try
                {
                    ...
                }
                catch
                {
                    disposable.Dispose();
                    throw;
                }
```

**Comment by giggio** [comment]
Agreed!

**Comment by giggio** [comment]
Closed on #631
